examples/webgl_loader_mmd.html

```diff
@@ -160,23 +172,76 @@
 
 				if( mesh ) {
 
-					mixer.update( clock.getDelta() );
-
-					/* Note: ikSolver.resetLinks() is workaround for r74dev.
-					 *       THREE.PropertyMixer.apply() seems to have buffer cache
-					 *       when mixer.update() is called.
-					 *       ikSolver.update() change bone position/quaternion
-					 *       without mixer.update() then buffer cache will be inconsistent.
-					 *       ikSolver.resetLinks() explicitly resets IK link bones quaternion and
-					 *       tries to avoid issues caused by this inconsistent.
-					 */
-					ikSolver.resetLinks();
+					var delta = clock.getDelta();
+					mixer.update( delta );
+
+					// see below for what this's for
+					backupBones( mesh );
+
 					ikSolver.update();
+					physics.update( delta );
 
 				}
 
 				renderer.render( scene, camera );
 
+				if ( mesh ) {
+
+					// see below for what this's for
+					restoreBones( mesh );
+
+				}
+
+			}
+
+			/*
+			 * Note: These following three functions are workaround for r74dev.
+			 *       THREE.PropertyMixer.apply() seems to save values into buffer cache
```

Comment:
In MMD system, IK and Physics calculate the final bone position/quaternion from pure skinning result in each frame. So backup/restore bones works.

Without the workaround the problem occurs when IK and Physics
change the bone position/quaternion while mixer.update() wants
to keep the same value at next frame.
mixer.update() decides not to update the value because the buffer cache isn't
changed though actual values are changed.